pib-rocks/pib-backend
# PR-622=>develop

# Jira Ticket:

Link to corresponding jira ticket:
https://pib-rocks.atlassian.net/jira/software/c/projects/PR/boards/3?selectedIssue=PR-622

# Checklist:
Necessary tasks before pull request can be merged:
- [x] All acceptance criteria have been implemented
- [x] The code has been reviewed and approved at least once
- [ ] All necessary tests have been successfully conducted
- [x] If necessary: A documentation covering the new concepts and changes has been created

# Testing:
### Testing approach for setups:
- [ ] Dev-Test


### Testing acceptance criteria:
- [ ] New audio files are used for turning the voice assistant on and off

### Testing instructions:
- turn the voice assistant on and off and check if the new audio files can be heard
